src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+   "source": [
+    "# source file for external flowsheets\n",
+    "from hda_flowsheets import hda_with_flash, hda_with_distillation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Building flowsheet...\n",
+      "\n",
+      "Setting inputs...\n",
+      "\n",
+      "Initializing flowsheet...\n",
+      "\n",
+      "Limiting Wegstein tear to 5 iterations to obtain initial solution, if not converged IPOPT will pick up and continue.\n",
+      "\n",
+      "WARNING: Wegstein failed to converge in 5 iterations\n",
+      "Solving flowsheet...\n",
+      "\n",
+      "Complete.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# alias 'm' for hda model with second flash unit\n",
+    "m = hda_with_flash(tee=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Building flowsheet...\n",
+      "\n",
+      "Setting inputs...\n",
+      "\n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_in[0.0].pressure\n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Trying to get scaling factor for unnamed expr \n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Trying to get scaling factor for unnamed expr \n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Trying to get scaling factor for unnamed expr \n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Trying to get scaling factor for unnamed expr \n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Trying to get scaling factor for unnamed expr \n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Trying to get scaling factor for unnamed expr \n",
+      "2022-07-21 10:56:57 [WARNING] idaes.core.util.scaling: Missing scaling factor for fs.H102.control_volume.properties_in[0.0].pressure\n",
+      "Initializing flowsheet...\n",
+      "\n",
+      "Limiting Wegstein tear to 3 iterations to obtain initial solution, if not converged IPOPT will pick up and continue.\n",
+      "\n",
+      "WARNING: Wegstein failed to converge in 3 iterations\n",
+      "Solving flowsheet...\n",
+      "\n",
+      "Adding distillation column and resolving flowsheet...\n",
+      "\n",
+      "2022-07-21 10:57:26 [WARNING] idaes.core.base.unit_model: DEPRECATION: <class 'idaes.core.base.process_block._ScalarTrayColumn'> has overloaded the initialize method. In v2.0, IDAES Will be moving to having a centralized initialize method which calls unit-specific initialize_build methods instead. Model developers should update their models to implement the initialize_build method instead of overloading initialize.\n",
```

Comment:
We might want to leave the distillation column version out for now - unless you really want to go and fix all these issues.

I suspect it would also be easier to follow for the user to have only one example (at least in the first pass) where you go through the steps. If we feel a comparison is really needed (and meaningful), you can do a quick build of the second example at the end through a script (which could also hide all this output).

Reply:
I believe the comparison of the two-flash and distillation economics is important and meaningful to include. I'll aim to fix the warnings first, and otherwise divert the output for that call so it doesn't print.